whatsapp.py

```diff
@@ -289,4 +133,3 @@ def main(path,name):
 if __name__ == "__main__":
     path = "/Users/mayam/Downloads/chats/WhatsApp Chat with "
```

Comment:
Do you change the path and name each time you run? seems uncomfortable. Would you prefer we'll find another way?

Reply:
Yeah, maybe we can just have a specific place in our project for placing chats in, would make much more sense

Reply:
let's add a todo, it's not for this PR